### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by Xevalous** [krisk/Fuse#706] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/83714022/205486675-ce596d3b-a674-4769-b72c-dbc13010b030.png)

i have object that accept all text but ends with 'menu' and the object that only accept 'menu'.
is there a method to handle this thing in fuse.js?

**Comment by krisk** [comment, marked as answer]
Late reply — sorry about that.

It's hard to tell exactly what you're after from the screenshot alone, but it sounds like you might want extended search. With `useExtendedSearch: true`, you can use the suffix operator (`$menu`) to match strings ending with "menu", or the exact-match operator (`=menu`) to match only the literal string "menu".

If you're still working on this and can share more details, happy to help.
